JS/projects.js

```diff
@@ -27,5 +27,5 @@ rawProject.forEach(function(projectObject){
 console.log(projects);
 
 projects.forEach(function(project){
-  $('.pro-list').append(project.toHtml());
```

Comment:
This probably wasn't working because the element you are trying to add to has a class of `proj_list` not `pro-list`. Just FYI.